tests/scheduler/test-SchedulerServer.cpp

```
@@ -0,0 +1,93 @@
+    REQUIRE(metadata_store->remove_job(job_id).success());
+    REQUIRE(res_buffer.has_value());
+    if (res_buffer.has_value()) {
+        msgpack::object_handle const handle
+                = msgpack::unpack(res_buffer.value().data(), res_buffer.value().size());
+        msgpack::object const object = handle.get();
+        spider::scheduler::ScheduleTaskResponse const res
+                = object.as<spider::scheduler::ScheduleTaskResponse>();
+        REQUIRE(res.has_task_id());
+        REQUIRE(res.get_task_id() == parent_task.get_id());
+    }
+    server.stop();
```

cppcheck (2.10-2)

[performance] Variable 'id' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)